src/theme/light-theme.tsx

```diff
@@ -55,84 +62,79 @@ import {
-    .map((_, i) => i + 1)
-    .map((i) => [0, i / 12, i, i / 4])
-    .map((v) => v.map(Math.ceil))
-    .map(
-      (v) =>
-        `${v[0]}px ${v[1]}px ${v[2]}px ${v[3]}px ${alpha(sb_dark_blue, 0.07)}`,
-    ),
+  '0px 4px 8px 0px rgba(27, 36, 63, 0.08)',
+  '0px 8px 24px 0px rgba(27, 36, 63, 0.08)',
+  '0px 16px 32px 0px rgba(27, 36, 63, 0.08)',
+  '0px 16px 48px 0px rgba(27, 36, 63, 0.08)',
+  ...Array(20).fill('none'),
 ] as Theme['shadows']
 
 const spacing = (factor?: number) => Math.floor(factor ?? 1) * 4
 
 const palette = {
   mode: 'light',
   primary: {
-    main: color_primary,
-    contrastText: white,
+    main: sb_primary,
+    contrastText: sb_neutral_white,
   },
   secondary: {
-    // main: color_secondary,
-    main: color_primary_dark,
-    contrastText: white,
+    main: sb_secondary,
+    contrastText: sb_neutral_white,
   },
   success: {
-    main: color_positive,
-    contrastText: white,
+    main: sb_success,
+    contrastText: sb_neutral_white,
   },
   info: {
-    main: blue,
-    contrastText: white,
+    main: sb_info,
+    contrastText: sb_neutral_white,
   },
   warning: {
-    main: color_warning,
-    contrastText: white,
+    main: sb_warning,
+    contrastText: sb_neutral_black,
   },
   error: {
-    main: color_negative,
-    contrastText: white,
+    main: sb_danger,
+    contrastText: sb_neutral_white,
   },
   background: {
-    default: white,
-    paper: white,
+    default: sb_neutral_white,
+    paper: sb_neutral_white,
   },
   common: {
-    white: white,
-    black: black,
+    white: sb_neutral_white,
+    black: sb_neutral_black,
   },
   text: {
-    primary: sb_dark_blue,
-    secondary: sb_dark_blue_50,
-    disabled: light_50,
+    primary: sb_info_950,
+    secondary: sb_base_700,
+    disabled: sb_base_500,
   },
-  divider: light,
+  divider: sb_base_200,
   action: {
-    focus: light_50, // background-color
+    focus: sb_base_500, // background-color
     focusOpacity: focusOpacity,
-    active: sb_dark_blue_50,
+    active: sb_base_700,
     activatedOpacity: activatedOpacity,
-    disabled: sb_dark_blue_25,
-    disabledBackground: light_25,
+    disabled: sb_secondary_50,
+    disabledBackground: sb_overlay_secondary,
     disabledOpacity: disabledOpacity,
-    hover: light_25, // background-color
+    hover: sb_secondary_50, // background-color
     hoverOpacity: hoverOpacity,
-    selected: light_50, // color
+    selected: sb_base_500, // color
     selectedOpacity: selectedOpacity,
   },
   grey: {
-    '50': '#f7f8f9',
-    '100': '#ECF0FF',
-    '200': '#E2E6F5',
-    '300': '#D1D5E4',
-    '400': '#ADB1C0',
-    '500': '#8d919f',
-    '600': '#656976',
-    '700': '#525662',
-    '800': '#343743',
-    '900': '#141822',
+    '50': sb_base_50,
+    '100': sb_base_100,
+    '200': sb_base_200,
+    '300': sb_base_300,
+    '400': sb_base_400,
+    '500': sb_base_500,
+    '600': sb_base_600,
+    '700': sb_base_700,
+    '800': sb_base_800,
+    '900': sb_base_900,
   },
```

Comment:
We could perhaps move this whole object to design-tokens/palette

Reply:
it's possible, just a point, in design tokens there's on more color (950) that we could not pass here since the mui only maps until 900

Reply:
@marcosh-ramos, the object could still contain it. For grey, we could create a new object where we just remove the 950

Reply:
done https://github.com/storyblok/mui/pull/48/commits/e66a71cf031abebc0d22ee84551b2357dd7379ef#diff-e6dca4a2cd646e67ccaaa1d6e49d8d017a71dbba22ba47fa69ad4ddb96119156R132